easybuild/easyconfigs/s/setuptools/setuptools-23.1.0-GCC-5.4.0-2.26-Python-2.7.12.eb

```diff
@@ -0,0 +1,32 @@
+easyblock = "PythonPackage"
+
+name = "setuptools"
+version = "23.1.0"
+
+homepage = "https://pypi.python.org/pypi/setuptools/"
+description = """Download, build, install, upgrade, and uninstall Python packages -- easily!"""
+
+toolchain = {'name': 'GCC', 'version': '5.4.0-2.26'}
+
+source_urls = [PYPI_SOURCE]
+sources = [SOURCE_TAR_GZ]
+
+checksums = ['4e269d36ba2313e6236f384b36eb97b3433cf99a16b94c74cca7eee2b311f2be']
+
+python = "Python"
+pythonversion = "2.7.12"
+pythonsuffix = "-bare"
+
+versionsuffix = "-%s-%s" % (python, pythonversion)
+
+dependencies = [(python, pythonversion, pythonsuffix)]
+
+py_short_ver = ".".join(pythonversion.split(".")[0:2])
+pylibdir = "lib/python%s/site-packages/%s" % (py_short_ver, name)
+
+sanity_check_paths = {
+    'files': ["bin/easy_install", "%s-%s-py%s.egg" % (pylibdir, version, py_short_ver)],
```

Comment:
we typically refrain from check the `.egg`, because it may be either a file (zipped egg) or a directory (unpacked egg)

instead, checking for `lib/python%(pyshortver)s/site-packages` as dir is sufficient